### PR TITLE
Fix line breaks

### PR DESCRIPTION
**Pull request by forki**
This is the current situation:

image

I hope line breaks are now ok

### CODE REVIEW

**Comment by haf** [comment]
Please write a test for this code change.

**Comment by forki** [comment]
will do!

**Comment by forki** [comment]
@haf @Krzysztof-Cieslak I'm pretty close, but how can I go from Message to String? ToString just gives me the type.

**Comment by forki** [comment]
Ok found a way by adding a helper function.
I also enabled FAKE's expecto test helper for dogfooding.

File: build.fsx [hunk 1]
```diff
@@ -53,6 +55,11 @@ Target "Build" (fun _ ->
   |> ignore
 )
 
+Target "RunTests" (fun _ ->
+    !! testExecutables
+    |> Expecto (fun p -> { p with Parallel = false } )
```
Comment (haf) [inline build.fsx]: I believe Expeco's own tests should be runnable in parallel. :)

**Pull request merged**